Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -49,26 +53,48 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
             // https://colinbarrebrisebois.com/2011/03/07/gdc-2011-approximating-translucency-for-a-fast-cheap-and-convincing-subsurface-scattering-look/
 
             {
-                thickness = max(shadowRatio, surface.transmission.thickness);
+                thickness = max(transmissionDistance, surface.transmission.thickness);
                 float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmissionParams.z ) ) ), transmissionParams.y ) * transmissionParams.w;
                 float lamberAttenuation = exp(-thickness * transmissionParams.x) * saturate(1.0 - thickness);
                 result = transmittance * lamberAttenuation * lightIntensity;
             }
             break;
 
         case TransmissionMode::ThinObject:
-            // Thin object mode, using thin-film assumption proposed by Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
+            // Thin object mode, based on Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
             // http://www.iryoku.com/translucency/downloads/Real-Time-Realistic-Skin-Translucency.pdf
-
-            float litRatio = 1.0 - shadowRatio;
-            if (litRatio)
+            
             {
-                const float thickness = surface.transmission.thickness * transmissionParams.w;
-                const float3 invScattering = rcp(transmissionParams.xyz);
-                const float falloff = ThinObjectFalloff(surface.normal, dirToLight);
-                result = TransmissionKernel(thickness, invScattering) * falloff * lightIntensity * litRatio;
-            }
+                // transmissionDistance < 0.0f means shadows are not enabled --> avoid unnecessary computation
+                if (transmissionDistance < 0.0f)
+                {
+                    break;
+                }
 
+                // Irradiance arround surface point. 
+                // Albedo at front (surface point) is used to approximate irradiance at the back of the object
+                // See observation 4 in [Jimenez J. et al, 2010] 
+                // Increase angle of influence (angle(N,L) -> angle(N,L) + acos(transmissionNdLBias)) to smooth transition regions
+                float3 E = surface.albedo * saturate(lightingData.transmissionNdLBias + dot(-surface.normal, dirToLight));
```

Comment:
Comment says 
angle(N,L) + acos(transmissionNdLBias))
but calculation is
cos(angle(N,L) ) + transmissionNdLBias
Slightly confusing, would either leave the comment out or elaborate on the difference between the two (or am I missing something and the math works out to be the same?)

Reply:
The comment referred to the actual angle values. Since NdL = cos(angle(N,L)), then the angle added due to the bias would be acos(transmissionNdLBias), which is added to angle(N,L) to account for the total (extended) angle.  
I agree it was a bit confusing, and since the instruction in code is already self-explanatory, I removed it (https://github.com/o3de/o3de/pull/6428/commits/024933d4a177997467dc97271610dcb5f450363b).